src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -134,6 +161,11 @@ Released 2022-Sep-29
   `AddOtlpExporter` extension to allow for more fine-grained options management
   ([#3653](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3653))
 
+* `OtlpExporterOptions` can now be bound to `IConfiguation` and
```

Comment:
This change was only for OtlpLogExporter, right? It would be good to mention that.

Reply:
Why put this change under `1.5.0-alpha.2` and not Unreleased?

Reply:
it is not under 1.5.0-alpha.2. It is under 1.4.0-beta1

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4556#discussion_r1222410410

Reply:
Updated - does this work?